### PR TITLE
feat: added modal component

### PR DESCRIPTION
**Pull request by shrushti2000**
### Added Modal component.
[click here to Watch live preview](https://jovial-almeida-56c2f9.netlify.app/adornui-documentation/modal)

ezgif com-gif-maker (6)

### CODE REVIEW

File: AdornUI-documentation/modal.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                <a class="sidebar-item-link" href="avatar.html">Avatar</a>
+                <a class="sidebar-item-link" href="card.html">Cards</a>
+                <a class="sidebar-item-link" href="button.html">Buttons</a>
+                <a class="sidebar-item-link" href="input.html">Input</a>
+                <a class="sidebar-item-link" href="modal.html">Modal</a>
+                <a class="sidebar-item-link" href="navigation.html">Navigation</a>
+            </div>
+        </aside>
+        <main>
+            <section class="component-doc-container">
```
Comment (mohitdhatrak) [inline AdornUI-documentation/modal.html:41]: good work using semantic elements

File: AdornUI-documentation/modal.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/x-icon" href="./images/adornui-logo.png">
+    <title>AdornUI</title>
```
Comment (mohitdhatrak) [inline AdornUI-documentation/modal.html]: just a suggestion, you can add meta tags with author name and page description

File: modal.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const openBtn = document.querySelector(".modal-open-btn");
+const closeBtn = document.querySelector(".modal-close-btn");
+var modal = document.querySelector(".modal-overlay");
```
Comment (mohitdhatrak) [inline modal.js]: any reason for using var after using const above?

File: AdornUI-Stylesheets/modal.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+.modal-content {
+  height: 80%;
+}
+.modal-footer {
+  padding: 0.5rem;
+  height: 20%;
+  justify-content: space-between;
+  align-items: center;
+  height: 5rem;
```
Comment (mohitdhatrak) [inline AdornUI-Stylesheets/modal.css]: height 20% is getting overridden by height 5rem you can remove it

File: AdornUI-Stylesheets/modal.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.4);
+  justify-content: center;
+  align-items: center;
+  visibility: hidden;
+}
+.modal-box {
```
Comment (mohitdhatrak) [inline AdornUI-Stylesheets/modal.css:12]: also a suggestion, for better readability add 1 line space after each CSS block

**Review by mohitdhatrak — COMMENTED** [review]
Few suggestions given, have a look. That's all from me, good work!

File: modal.js (hunk 3 above)
Comment (shrushti2000) [reply modal.js]: No, sorry that was by mistake. Thanks for bringing that to notice. Will change it soon to const.

**Comment by shrushti2000** [comment]
Thanks @mohitdhatrak for reviewing the PR. Will make the suggested changes soon.

**Review by ark2002 — CHANGES_REQUESTED** [review]
other than changes suggested by mohit everything looks fine

File: modal.js (hunk 3 above)
Comment (mohitdhatrak) [reply modal.js]: Np

**Pull request merged**